### PR TITLE
Card 25 passes tests. Still needs to deploy to heroku.

### PR DESCRIPTION
**Pull request by CoryBethune**
User Story:

As a merchant
When I visit my merchant invoice show page
Then I see all of my items on the invoice including:

Item name
The quantity of the item ordered
The price the Item sold for
The Invoice Item status
And I do not see any information related to Items for other merchants

File edited:
app/controllers/invoices_controller.rb
app/views/invoices/show.html.erb
config/routes.rb
spec/features/invoices/show_spec.rb

Useful info: N/A

Additional comment: PLEASE manually edit the .ruby-version, Gemfile, and Gemfile.lock so that the changes I added are NOT merged into main. Please erase my edits to these files so that y'all do not have to manually update them when you pull down from main once this is merged.

### CODE REVIEW

**Comment by CoryBethune** [comment]
Ok, ignore the comment I put in the "Additional comment" section. I was able to modify those files once I created the PR so y'all don't have to worry with it.

**Comment by AliciaWatt** [comment]
pulled it down one rspec test failed because the branch is not updated with what's on main. Should be able to resolve this issue in merge conflicts.

File: app/controllers/invoices_controller.rb [hunk 1]
```diff
@@ -7,6 +7,8 @@ def index
 
   def show
     @invoice = Invoice.find(params[:invoice_id])
+    @items = @invoice.items
```
Comment (AliciaWatt) [inline app/controllers/invoices_controller.rb:10]: great for now! Maybe in the future, we work this down to just one instance variable

File: app/controllers/invoices_controller.rb [hunk 2]
```diff
@@ -7,6 +7,8 @@ def index
...
+    # binding.pry
```
Comment (AliciaWatt) [inline app/controllers/invoices_controller.rb:11]: remove pry.  just trying to bulk up this comment section

File: spec/features/invoices/show_spec.rb [hunk 3]
```diff
@@ -28,4 +31,16 @@
...
+  it "display invoice item information on the invoice show page" do
+    visit "/merchants/#{@merchant1.id}/invoices/#{@invoice3.id}"
+
+    expect(page).to have_content(@invoice_item4.quantity)
+    expect(page).to have_content(@invoice_item4.unit_price)
+    expect(page).to have_content(@invoice_item4.status)
+    expect(page).to have_content(@item4.name)
+    expect(page).to have_no_content(@item3.name)
+    expect(page).to have_no_content(@item2.name)
+    expect(page).to have_no_content(@item1.name)
```
Comment (AliciaWatt) [inline spec/features/invoices/show_spec.rb:43]: nice use of the sad path! Sad paths make us happy!

**Pull request merged**